### DISCUSSION TITLE
相手のHPが減らないです

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#140] [Question-original]
## (1) 実現したいこと
テクモンの完成。相手HPが減らない。
## (2) 実現するために自分が試した内容とその結果
テクモン30ページまでやりました。途中までは相手HPが減るのも確認できていましたが、最後まで実装したら相手HPが100で固定され減らなくなってしまいました。
## (3) 調べた内容や資料
教科書009テクモン
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
不明です。enemyHP関連で色々みたのですが、スクリプトの量が多く原因を突き止められませんでした。
## (5) GitHubのリンク
https://github.com/KOTARO1209/TechMon

**Comment by txxxxc** [comment]
ここらへんがCharacterクラスのHP,MPじゃなくて、ViewControllerに定義してるHP, MPを参照してるからっぽいので、確認してみて下さいー！
https://github.com/KOTARO1209/TechMon/blob/d456c1226f8d74319afd8126eecdbb076f12d8f2/TechMon/BattleViewController.swift#L65-L82
